src/pybullet_blocks/envs/clear_and_place_env.py

```diff
@@ -294,14 +294,13 @@ def reset(
         scene_description = self.scene_description
         assert isinstance(scene_description, ClearAndPlaceSceneDescription)
 
-        # Place target area
-        target_position = self.np_random.uniform(
-            self.scene_description.target_init_position_lower,
-            self.scene_description.target_init_position_upper,
-        )
-        set_pose(
-            self.target_area_id, Pose(tuple(target_position)), self.physics_client_id
+        # Place target area at fixed position
+        target_position = (
```

Comment:
can we derive these values in terms of the scene description? (in case the scene description changes later we don't want the test to break)